File: src/main/java/com/rtomyj/skc/enums/Rarity.java
package com.rtomyj.skc.enums;


public enum Rarity
{

	C("Common")
	, R("Rarity")
	, SR("Super Rare")
	, UR("Ultra Rare")
	, SCRT("Secret Rare");

	private final String rarityName;

	Rarity(final String rarityName) { this.rarityName = rarityName; }

	@Override
	public String toString()
	{
		return rarityName;
	}

}

File: src/main/kotlin/com/rtomyj/skc/util/enumeration/Rarity.kt
package com.rtomyj.skc.util.enumeration

enum class Rarity(private val rarityName: String) {
  C("Common"),
  R("Rarity"),
  SR("Super Rare"),
  UR("Ultra Rare"),
  SCRT("Secret Rare");


  override fun toString(): String = rarityName
}